components/actiongroup/index.css

```diff
@@ -114,51 +108,40 @@ governing permissions and limitations under the License.
         z-index: 3;
```

Comment:
I think that &:focus-ring is supposed to be &:focus-visible.

It looks like the intent of the focus behavior on this component is to remove border radius from the corner of the focus ring where compact items connect.

This isn’t working because the focus ring is applied to action button as a box-shadow on the :after element. The focus ring on action button also lacks the focus ring gap, making the focus state almost invisible on emphasized buttons. 

I think the focus ring on action button should be applied as a border instead of a box shadow. Then the border radii of the compact action group items can changed per item.

Reply:
Thanks @lunarfusion — I think this is a larger question around how we're handling focus rings, but the implementation here is correct.